Lib/http/cookiejar.py

```diff
@@ -2107,7 +2121,9 @@ def save(self, filename=None, ignore_discard=False, ignore_expires=False):
                 else:
                     name = cookie.name
                     value = cookie.value
+                if cookie.httponly: domain = '#HttpOnly_' + cookie.domain
+                else: domain = cookie.domain
```

Comment:
Please do not put multiple statements on the same line (PEP 8).

Reply:
Fine with me, but note that in this case I am trying to emulate the "preexisting style" of this module.

You want me to change the PR to adhere to the modern PEP-8 guidelines in the added/modified lines, even though the rest of the file doesn't?

Reply:
> You want me to change the PR to adhere to the modern PEP-8 guidelines in the added/modified lines, even though the rest of the file doesn't?

Done in latest version.